### ISSUE TITLE
2010 condition data in /cvmfs/cms-opendata-conddb.cern.ch

### ISSUE THREAD

**Issue by katilp** [cernopendata/opendata.cern.ch#1549]
The condition data for 2010 have now been written in

for Public data 2010RunB: runs 146428 -149294
and All 2010: runs 136035 -149294

GlobalTag FT_R_42_V10A::All

The new VM #1298 will be updated so that it can access /cvmfs/cms-opendata-conddb.cern.ch

**Comment by katilp** [comment]
Testing the condition data reading with CMS-2010-OpenData-v4 (check with Jakob that this is the most recent wrt those mentioned in #1298)
- 2010 example from https://github.com/ayrodrig/pattuples2010 with
```
CMS Shell > ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A FT_R_42_V10A
CMS Shell > ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A.db FT_R_42_V10A.db
```
and 

```
#globaltag
#process.GlobalTag.globaltag = 'GR_R_42_V25::All' 
process.GlobalTag.connect = cms.string('sqlite_file:/cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A.db')
process.GlobalTag.globaltag = 'FT_R_42_V10A::All'
```


Start 14:41 (VM time).
with 
cvmfs2                 4096000    914977   3181024  23%

**Comment by katilp** [comment]
NB the CMS shell of the VM has 1 hour time difference to the normal shell 😕

**Comment by katilp** [comment]
Testing with https://cernbox.cern.ch/index.php/s/L8nXVBYoLXwqbbJ (CMS 2010 OpenData CernVM 4 v3)

As above,
but starts with 15:36 (NB in the case above, tried reading with direct process.GlobalTag.globaltag = 'GR_R_42_V25::All' without ln's first - killed in a similar way- , may explain the difference in cache occupation)
cvmfs2                     4096000  613195   3482806  15%

**Comment by katilp** [comment]
Instruction summary to reproduce the problem (in the CMS shell of the VM):
cmsrel CMSSW_4_2_8
cd CMSSW_4_2_8/src
cmsenv
git clone https://github.com/ayrodrig/pattuples2010.git 
cd pattuples2010
ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A FT_R_42_V10A
ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A.db FT_R_42_V10A.db

Check if /cvmfs/cms-opendata-conddb.ch is visible if needed
ls /cvmfs/cms-opendata-conddb.cern.ch

In PAT_data_repo.py replace `process.GlobalTag.globaltag = 'GR_R_42_V25::All' ` with
process.GlobalTag.connect = cms.string('sqlite_file:/cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A.db')
process.GlobalTag.globaltag = 'FT_R_42_V10A::All'

cmsRun PAT_data_repo.py

**Comment by katilp** [comment]
Increase the memory to 2 GB. Problem solved. However, even with condition data in the cache, the jobs takes long to start. Check if it depends on time/place.

**Comment by katilp** [comment]
Final testing with 
- [ ] reading condition data from frontier (no ln's, no mod's to config file
- [ ] reading condition data from cvmfs

**Comment by katilp** [comment]
Data reading is fine and tested, add the files (FT_R_42_V10A.db and zip of FT_R_42_V10A) to the record
http://opendata.cern.ch/record/1800.

Add in the description:

`<p> These files are needed when running a CMSSW job on the Open Data VM. They are automatically read from the /cvmfs/cms-opendata-conddb.cern.ch runtime, you do not need to download these files from here separately.</p>\n      <p>FT_R_42_V10A  is to be used with the collision data of RunB 2010 (with the event range of the open data.</p>\n      <p>See further information in <a href=\"/docs/cms-guide-for-condition-database\">the guide to CMS condition database</a>.</p> \n

`

**Comment by katilp** [comment]
Finalize this, and add the condition data for 2010 MC as well, see #2339 
(Samples produces with Release: CMSSW_4_2_8_lowpupatch1 Global Tag: START42_V17B::All)

**Comment by katilp** [comment]
To do  for FT_R_42_V10A:
- update the existing record http://opendata.cern.ch/record/1800. as in the comment (https://github.com/cernopendata/opendata.cern.ch/issues/1549#issuecomment-364145543) above
  - [ ] add the zipped db file to the record
  - [ ] add the text to the record

To do GT START42_V17B:
- [ ] create the corresponding record (similar to http://opendata.cern.ch/record/1804) with name
Condition data for 2010 simulated datasets
- [ ] add the zipped db file to the record

- [ ] update the docs in http://opendata.cern.ch/docs/cms-guide-for-condition-database correspondingly

**Comment by katilp** [comment]
Closing as addressed in #2670 (records created, decided not to add the zipped db file)
